actions/stack/get-usns/entrypoint/main.go

```diff
@@ -76,6 +82,18 @@ func main() {
 
 	flag.Parse()
 
+	distroExists := distroToVersionRegex[config.Distro]
+	if distroExists == "" {
+		var validDistroValues = ""
+
+		for key := range distroToVersionRegex {
+			validDistroValues = validDistroValues + key + "\n"
+		}
+
+		errMessage := fmt.Sprintf("--distro flag has to be one of the following values: \n %s", validDistroValues)
+		log.Fatal(errMessage)
+	}
```

Comment:
```suggestion
	_, ok := distroToVersionRegex[config.Distro]
	if !ok {
		log.Fatal(fmt.Sprintf("--distro flag has to be one of the following values: %v", slices.Sorted(maps.Keys(distroToVersionRegex))))
	}
```

Just a suggestion, these features are relatively new to the language and require a bump of the `go.mod`.